astropy/cosmology/core.py

```diff
@@ -328,9 +332,12 @@ def __equiv__(self, other: Any, /) -> bool:
 
         # Check all parameters in 'other' match those in 'self' and 'other' has
         # no extra parameters (latter part should never happen b/c same class)
-        params_eq = (set(self.__all_parameters__) == set(other.__all_parameters__)
-                     and all(np.all(getattr(self, k) == getattr(other, k))
-                             for k in self.__all_parameters__))
+        params_eq = set(self.__all_parameters__) == set(
+            other.__all_parameters__
+        ) and all(
+            np.all(getattr(self, k) == getattr(other, k))
+            for k in self.__all_parameters__
+        )
```

Comment:
```suggestion
        params_eq = (
            (set(self.__all_parameters__) == set(other.__all_parameters__)
            and all(
                np.all(getattr(self, k) == getattr(other, k)) for k in self.__all_parameters__
            )
        )
```

Does this pass black?

Reply:
I think there is a missing `)` in there and when I add that, black corrects it back to the original.

However, if one looks one line below (line 341 in this case) we find we just return `params_eq` so if I replace `param_eq` with `return` and remove the original `return` we get:

```python
        return set(self.__all_parameters__) == set(other.__all_parameters__) and all(
            np.all(getattr(self, k) == getattr(other, k))
            for k in self.__all_parameters__
        )
```
after black formats the file. Is this better?

Reply:
This does look better!